db/1.9/modules/ROOT/nav.adoc

```diff
@@ -6,6 +6,7 @@
 ** xref:database-connector-connection.adoc[Configure a Database Connection]
```

Comment:
edit instructions

Type: Copyedit
Content: All green
Due date: by next Tuesday 9th is fine.
Context: These are more chunks of configurations of the operation additional parameters, are not an example from start to finish like we would ideally love to have for every connector.